setup.cfg

```diff
@@ -0,0 +1,48 @@
+[metadata]
+name = geniesp
+version = attr: geniesp.__version__
+description = Export cBioPortal files for BPC
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/Sage-Bionetworks/GENIE-Sponsored-Projects
+author = Thomas Yu
+author_email = thomas.yu@sagebionetworks.org
+license = MIT
+license_file = LICENSE
+platforms = any
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+project_urls =
+    Bug Tracker = https://github.com/Sage-Bionetworks/synapsemonitor/issues
+    Source Code = https://github.com/Sage-Bionetworks/synapsemonitor
+
+[options]
+packages = find:
+install_requires =
+    synapseclient>=2.5.1
+    aacrgenie>=12.5.0
+    pandas>=1.0
+python_requires = >=3.7, <3.10
```

Comment:
just fyi, default python version on service catalog instance for RStudio is 3.6.9, which caused installation to fail initially.  Is there a reason to keep >=3.7 vs >=3.6?

Reply:
Python 3.6 is end of life. We should look to update the python on the rstudio instance.